ci/build_wheel.sh

```diff
@@ -16,5 +16,5 @@ sed -i -E "s/^name = \"${package_name}(.*)?\"$/name = \"${package_name}${PACKAGE
 
 python -m pip wheel "${package_dir}"/ -w "${package_dir}"/dist -vvv --no-deps --disable-pip-version-check
 
-python -m auditwheel repair -w ${package_dir}/final_dist --exclude "libcuda.so.1" --exclude "libnvidia-ml.so.1" ${package_dir}/dist/*
-RAPIDS_PY_WHEEL_NAME="ucx_${RAPIDS_PY_CUDA_SUFFIX}" rapids-upload-wheels-to-s3 cpp ${package_dir}/final_dist
+python -m auditwheel repair -w ${package_dir}/final_dist --exclude "libcuda.so.1" --exclude "libnvidia-ml.so.1" --exclude "libucm.so.0" --exclude "libuct.so.0" --exclude "libucs.so.0" --exclude "libucp.so.0" ${package_dir}/dist/*
```

Comment:
After all these exclusions, is there anything left to repair or can the entire line be removed?

Reply:
Yes, we are still bundling libcudart, libnuma, and libgomp. We'll probably want to think a bit more carefully about these (especially the latter two) in the near future, but I don't want to bottleneck current progress on analyzing what exactly we bundle.